tox.ini

```
@@ -28,6 +28,11 @@ commands = flake8
 commands = {posargs}
 passenv = *
 
+[testenv:functional]
+passenv = OS_*
+commands = pytest esiclient/tests/functional {posargs}
+VENV_NAME={envname}
```

I'm pretty sure this line needs to be:
```suggestion
setenv = VENV_NAME={envname}
```
for the env var to be set by tox